anomalib/config/config.py

```diff
@@ -177,7 +177,8 @@ def get_configurable_parameters(
     config = update_input_size_config(config)
 
     # Project Configs
-    project_path = Path(config.project.path) / config.model.name / config.dataset.name / config.dataset.category
+    category = config.dataset.category if "category" in config.dataset.keys() else ""
```

Comment:
Maybe it would be a bit more clear if we check the dataset type here, and only add the category to the path if the type is MVTec.